runtime/cudaq/builder/kernel_builder.h

```diff
@@ -551,6 +551,41 @@ class kernel_builder : public details::kernel_builder_base {
     const std::vector<QuakeValue> &qubits{first, second};
     details::swap(*opBuilder, empty, qubits);
   }
+  /// @brief SWAP operation for performing a Fredkin gate between two qubits,
+  /// based on the state of input `control` qubit/s.
+  template <typename mod, typename = typename std::enable_if_t<
+                              std::is_same_v<mod, cudaq::ctrl>>>
+  void swap(const QuakeValue &control, const QuakeValue &first,
```

Comment:
Up above in py_kernel_builder, the control was a `QuakeValue` rather than a const reference. Should it be a `const QuakeValue&` in both places?